VRDR/DeathRecord.cs

```diff
@@ -519,11 +519,9 @@ public string StateLocalIdentifier
         {
             get
             {
-                if (StateDocumentReference != null && StateDocumentReference.Identifier != null && StateDocumentReference.Identifier.Count() > 0)
-                {
-                    return StateDocumentReference.Identifier[0].Value;
-                }
-                return null;
+                // return first non-null/empty identifier value or null if non found
```

Comment:
Tiny typo: `non` -> `none` or perhaps `not`

Reply:
Not a typo, non is a valid prefix meaning not.

Reply:
I was referring to the second non:

> return first non-null/empty identifier value or null if **non** found

Reply:
Ah, in that case I agree, it should be none. Will fix.